File: src/PE1.go
package main
import "fmt"
func main() {
 var i = 0
 var sum = 0
 for i<= 999 {
    if i%3 == 0 || i%5 == 0  {
      sum = sum + i
    }
    i = i + 1
 }
 fmt.Println(sum)
}

File: src/rust/PE1.rs
fn main() {
    let mut i = 1;
    let mut sum = 0;
    while i < 1000
    {
     if i % 3 == 0 || i % 5 == 0 {
      sum = sum + i;
     }
     i = i + 1;
    }
    println!("{}", sum);
}

